pkg/proxy/ipvs/proxier.go

```diff
@@ -453,6 +457,63 @@ func NewProxier(ipt utiliptables.Interface,
 	return proxier, nil
 }
 
+// Create the "other" proxier for dual-stack
+func NewOtherProxyServer(
+	config *proxyconfigapi.KubeProxyConfiguration) (*Proxier, error) {
+
+	// TODO: Remove hard-coded ipv6
+	nodeIP := net.ParseIP("::1")
+	protocol := utiliptables.ProtocolIpv6
+	clusterCIDR := os.Getenv("CLUSTER_IPV6_CIDR")
```

Comment:
This is passed in config type, right?

Reply:
Yes but the necessary support is in PR https://github.com/kubernetes/kubernetes/pull/79576 .

I made this as a *temporary* configuration to keep my development env managable. It is *only* the `nodeIP` and `clusterCIDR` that are needed. With this compromise my dev env is really clean;

* I build k8s from your dual-tack PR branch
* I build only `kube-proxy` from my PR branch which is kept ip-to-date with k8s master

I am unsure how to handle this in another way. My plan was to wait until https://github.com/kubernetes/kubernetes/pull/79576 is merged to "master" and then make the (small) final update.

An alternative may be to cherry-pick the config commits from https://github.com/kubernetes/kubernetes/pull/79576.